### PR TITLE
Add site page to admin client

### PR DESCRIPTION
**Pull request by apburnes**
When an admin goes to a site page in the admin client, they will have access to all of the attribute data and builds for that site.

## Acceptance Criteria
- [x] Add 'site/:id' to page routes
- [x] Slim down sites list for easier viewing
- [x] View site attibutes
- [x] View site users
- [x] View builds

### CODE REVIEW

File: admin-client/src/components/LabeledItem.svelte [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<script>
+  export let label = 'label';
```
Comment (amirbey) [inline admin-client/src/components/LabeledItem.svelte:2]: does this value get redefined?

File: admin-client/src/components/LabeledItem.svelte [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  export let value = null;
```
Comment (amirbey) [inline admin-client/src/components/LabeledItem.svelte:3]: does this value get redefined?

File: admin-client/src/components/ExternalLink.svelte [hunk 3]
```diff
@@ -24,7 +25,7 @@
 <a
   rel="noreferrer"
   target="_blank"
-  class="font-code-2xs padding-x-1 text-primary text-ls-neg-3 text-uppercase"
+  class="font-code-2xs text-primary text-ls-neg-3 text-uppercase {className}"
```
Comment (amirbey) [inline admin-client/src/components/ExternalLink.svelte:28]: does this value get redefined?

File: admin-client/src/components/UserList.svelte [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+<SectionHeader>
+  Users
+</SectionHeader>
+<div class="grid-row grid-gap">
+  {#each users as user}
+    <div class="grid-col-12 tablet:grid-col-auto margin-y-05">
+      <div class="padding-x-1 padding-y-05 bg-gray-5">
+        <LabeledItem label="username" value={user.username} />
+        <LabeledItem label="email" value={user.email} />
+      </div>
```
Comment (amirbey) [inline admin-client/src/components/UserList.svelte:18]:
```suggestion
        <LabeledItem label="signed in" value={user.signedInAt} />
      </div>
```

File: admin-client/src/components/LabeledItem.svelte (hunk 2 above)
Comment (apburnes) [reply admin-client/src/components/LabeledItem.svelte:3]: Ya, that's the svelte way of creating a default value for a property.  Any variable in a component with the `export` syntax is a value you can pass to a component and the passed value will take precedent in the prop. 

ie
```svelte
// passing a value 
<LabeledItem label={'Hello'} value={'World'} />

// Shorthand version
<script>
  const label = 'Hello';
  const value = 'World';
</script>

<LabelItem {label} {value} />
```

File: admin-client/src/components/UserList.svelte (hunk 4 above)
Comment (apburnes) [reply admin-client/src/components/UserList.svelte:18]: 👍

File: admin-client/src/components/ExternalLink.svelte (hunk 3 above)
Comment (apburnes) [reply admin-client/src/components/ExternalLink.svelte:28]: Yup. Playing with the idea that you can add additional classes to the anchor for customization.  Using it with the '<SiteCard />' to add padding to the center external link.

**Pull request merged**